start.sh

```diff
@@ -1,4 +1,4 @@
-# docker run -d --name neo4j -e NEO4J_AUTH=neo4j/password -p 7687:7687 -p 7474:7474 neo4j:latest
-# docker ps -qaf name=neo4j
+docker run -d --name neo4j -e NEO4J_AUTH=neo4j/password -p 7687:7687 -p 7474:7474 neo4j:latest
+docker ps -qaf name=neo4j
 
-docker compose up --build --force-recreate
+docker-compose up --build --force-recreate
```

Comment:
Isn't `docker-compose` deprecated for `docker compose` ?

Reply:
"docker compose" is the updated version from Docker, yes. However, in the latest version of Debian GNU/Linux and the Unstable branch, it is still a separate command for "docker" and "docker-compose". That is why I suggested, if possible, to create an extra profile, e.g. --profile debianunstable, rather than assume every Linux can use --profile linux. I do not know if this is possible, but it would be nice. As a general rule, "docker compose", with the space is the updated way. I hope this helps.

Reply:
What I meant is we should encourage the use of `docker compose`. But if it is available on Debian Unstable, why not use it ?

Anyway, the changes you made should work even when using `docker compose` (the `context: .` is implicit in `docker compose`, don't know for `docker-compose`). The only thing I would do differently is creating another start script for the old `docker-compose` instead of changing the actual one.

I would still be interested in what error you had before, maybe there is another workaround!